test/metabase/test/data/interface.clj

```diff
@@ -364,6 +364,14 @@
 
 (defmethod supports-time-type? ::test-extensions [_driver] true)
 
+(defmulti supports-timestamptz-type?
+  "Whether this database supports a `timestamp with time zone` data type or equivalent."
+  {:arglists '([driver])}
+  dispatch-on-driver-with-test-extensions
+  :hierarchy #'driver/hierarchy)
```

Comment:
You've this multimethod returns false for athena, but there must be more drivers where this should return false. `sqlite` comes to mind. 

for another PR: is it possible to write a test that fails if the multimethod's value is incorrect for each driver?